### PR TITLE
highlight (H)EEx in guides

### PR DESCRIPTION
**Pull request by tmbb**
Highlight guides with the new (H)EEx makeup lexers.

### CODE REVIEW

**Comment by 03juan** [comment]
Some code blocks are tagged `html`, others `heex`. Is there a reason for the inconsistency?

**Comment by tmbb** [comment]
> Is there a reason for the inconsistency?

Yes, there is. Although HTML is of course valid HEEx, you get better syntax highlighting in editors if you tag a code block as HTML instead of HEEx.

The ones I've tagged HTML contain actual raw HTML instead of HEEx (even if they might be a static part of an HEEx template)

**Comment by 03juan** [comment]
Oh I see, I should have tested this on my side perhaps instead. Thank you for the clarification. 👍

File: guides/server/live-navigation.md [hunk 1]
```diff
@@ -78,7 +80,9 @@ the system and you define it in the router as:
 
 Now to add live sorting, you could do:
 
+```heex
     <%= live_patch "Sort by name", to: Routes.live_path(@socket, UserTable, %{sort_by: "name"}) %>
```
Comment (josevalim) [inline guides/server/live-navigation.md:84]:
```suggestion
<%= live_patch "Sort by name", to: Routes.live_path(@socket, UserTable, %{sort_by: "name"}) %>
```

**Review by josevalim — COMMENTED** [review]
This is glorious, thank you!

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**